AK/StdLibExtraDetails.h

```diff
@@ -602,6 +602,9 @@ struct EquivalentFunctionTypeImpl<T (C::*)(Args...) const> {
 template<typename Callable>
 using EquivalentFunctionType = typename EquivalentFunctionTypeImpl<Callable>::Type;
 
+template<typename T, typename... Args>
+inline constexpr auto IsAnyOf = (IsSame<T, Args> || ...);
+
```

Comment:
We already have `IsOneOf`, although `Any` sounds nicer and is closer to other library functions
So maybe a separate commit renaming it

Reply:
oh that's why I couldn't find it lol